packages/manager/modules/pci/src/projects/new/payment/controller.js

```diff
@@ -361,6 +366,21 @@ export default class PciProjectNewPaymentCtrl {
   }
 
   trackOnPaymentFormSubmit(tag) {
+    if (this.$cookies.get(MANAGER_TRACKING) === '1') {
```

Comment:
and if cookies is undefined ?

Reply:
Nothing happens. It's mentioned in the ticket to track the creation only if the user accepts the cookies policy